test/e2e/policies.test-suite/check-correct-policy.source.sh

```diff
@@ -0,0 +1,13 @@
+# This script does a policy check before the real test code is started.
+
+cache_policy="$(vm-command-q "cat /var/lib/cri-resmgr/cache" | jq -r .PolicyName)"
```

Comment:
hmm, shouldn't we escape inner " or replace them by '

Reply:
Answering to myself, the code works with current inner "" too, no need to change it.

Reply:
Yes, bash understands nested combinations of `" $( " $( " ... " ) " ) "`.